auth/auth_test.go

```diff
@@ -13,10 +13,13 @@ import (
 	loom "github.com/loomnetwork/go-loom"
 	"github.com/loomnetwork/go-loom/auth"
 	"github.com/loomnetwork/loomchain"
-	"github.com/loomnetwork/loomchain/features"
 	"github.com/loomnetwork/loomchain/store"
 )
 
+const (
+	IncrementNonceOnFailedTx = "NonceHandler.IncNonceOnFailedTx"
```

Comment:
`IncrementNonceOnFailedTx` -> `incrementNonceOnFailedTx`
No need to export this from a test.

Reply:
Fixed